### DISCUSSION TITLE
Welcome to pix_driver Discussions!

### DISCUSSION THREAD

**Discussion by ahuazuipiaoliang** [pixmoving-moveit/pix_driver#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by aemilio-Developer** [comment]
Hello! From the structure you gave to the vehicle interface for the pix_hook vehicle, I was wondering how it came to be integrated into autoware to be then later built and executed. Specifically, in the conversion part I implemented I have an include to an .hpp file that contains the control message coming from autoware, and whenever I try to build the package it gives me the following error: "fatal error: autoware_control_msgs/msg/control.hpp: No such file or directory
   12 | #include <autoware_control_msgs/msg/control.hpp>." How is this dependency resolved?
